apps/system/js/sync_manager.js

```diff
@@ -199,12 +210,13 @@
             return;
           }
           this.getAccount().then(account => {
+            this.user = account ? account.email : null;
```

Comment:
maybe `getAccount` should `this.user = account ? account.email : null;` before resolving. Looks like we're repeating it on each call site.